Ports/mrsh/package.sh

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=mrsh
 version=cd3c3a48055ab4085d83f149ff4b4feba40b40cb
-files="https://codeload.github.com/emersion/mrsh/legacy.tar.gz/${version} emersion-mrsh-d9763a3.tar.gz 6896493a1020774715ccca28e8d8f4ec722af63a93543fb6dd2762f7b1de9c8a"
+files="https://github.com/emersion/mrsh/archive/${version}.tar.gz ${port}-${version}.tar.gz d26e3fdee71ef168cf3f8ad2912c148b20aab524048e4ea899d6b83fb299ceab"
 auth_type=sha256
 useconfigure=true
-workdir=emersion-mrsh-d9763a3
 
-export CFLAGS=-Wno-deprecated-declarations
+pre_configure() {
+    export CFLAGS=-Wno-deprecated-declarations
+}
+
+post_configure() {
+    unset CFLAGS
+}
```

Comment:
Is this actually needed? Why can't we just keep the general `export CFLAGS=...` as before?

Reply:
This is not needed, but I've seen other packages do it this way, and I thought it was cleaner. If you say it is unneeded I'll revert to how it was.

Reply:
I've deleted the extra code. It wasn't doing anything usefull.